src/dom_components/view/ToolbarButtonView.js

```diff
@@ -22,9 +22,32 @@ export default Backbone.View.extend({
   handleClick(event) {
     event.preventDefault();
     event.stopPropagation();
-    const { em } = this;
```

Comment:
I think you forgot this 😬

Reply:
Oh right of course.
Can you explain what's the different between `this.em` and `this.editor`?

Reply:
`editor` is more or less a public API layer of `em` but they both act as a Service Manager

Reply:
In that case I don't like the fact that in this patch I'm using both `em` and `editor`. I think we should use either one not both.
You agree with that? Should I update my branch or something?